### ISSUE TITLE
Download Module for Gigaset

### ISSUE THREAD

**Issue by FendVario1** [amosproj/amos2022ss01-firmware-downloader#82]
## User story
1. As a user
2. I want / need a downloader for Gigaset firmware files
3. So that all firmware images can be downloaded automatically

## Acceptance criteria
* Automated Download of X Firmware images 
* Testcase checking image download is working
* Metadata & image is correctly added to file system/database

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
